drivers/hud_apr/app/controllers/hud_apr/base_controller.rb

```diff
@@ -53,13 +53,13 @@ def filter_params
         if options.present?
           @filter.start = options['start'].presence || Date.new(year - 1, 10, 1)
           @filter.end = options['end'].presence || Date.new(year, 9, 30)
-          @filter.coc_code = options['coc_code']
+          @filter.coc_codes = options['coc_codes'].presence || GrdaWarehouse::Config.get(:site_coc_codes)
```

Comment:
Careful here: `GrdaWarehouse::Config.get(:site_coc_codes)` returns a string despite it's name.